meta-sokol-flex-staging/files/toolchain-shar-extract.sh

```diff
@@ -1,16 +1,13 @@
 #!/bin/sh
 
-# ---------------------------------------------------------------------------------------------------------------------
-# SPDX-License-Identifier: MIT
-# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
Don't we still need SPDX License Identifier

Reply:
This is a tough one, we're supposed to as a matter of policy, but this also isn't a file we maintain, it's copied from upstream, so adding copyrights or license info to others files seems a bit odd to me. It's a good point, though.

Reply:
Fixed